app/views/feedback/create.html.haml

```diff
@@ -0,0 +1,3 @@
+Thanks for your feedback
+
+= link_to "Return to gov.au", @feedback.url
```

Comment:
I'm actually waiting on a bunch of content to go on this page. Once I have more detail I'll open another ticket and either add it or do as you suggest.